jest/preprocessor.js

```diff
@@ -14,6 +14,18 @@ module.exports = {
       }
       // Run our modules through Babel before running tests
       if (babel.util.canCompile(filename)) {
+        // This will match:
+        // import IconFoo from '../../icons/icon-medium/pages-code.svg?name=foo';
+        var matches = src.match(/(\w+) (.*?) (\w+)(.*?)(icons)(.*?)\.svg(\?.*?)$/img);
```

Comment:
@jfurrow could you check if this works too:
```
var matches = src.match(/^(\w+) (.*?) (\w+|=) '(.*?)(icons)(.*?)\.svg(\?.*?)';$/img);
```
I want to make sure we don't match other things on accident.